src/passes/stringify-walker.h

```diff
@@ -234,7 +234,11 @@ struct HashStringifyWalker : public StringifyWalker<HashStringifyWalker> {
 // Functions that filter vectors of SuffixTree::RepeatedSubstring
 struct StringifyProcessor {
   static std::vector<SuffixTree::RepeatedSubstring>
-  dedupe(const std::vector<SuffixTree::RepeatedSubstring>);
+  dedupe(const std::vector<SuffixTree::RepeatedSubstring> substrings);
```

Comment:
Since these functions consume their input and transform it into their output, it probably makes sense for them to take `std::vector<...>&&`.

Reply:
done!